tests/kafkatest/services/kafka_log4j_appender.py

```diff
@@ -32,14 +34,15 @@ def __init__(self, context, num_nodes, kafka, topic, max_messages=-1):
         self.max_messages = max_messages
 
     def _worker(self, idx, node):
-        cmd = self.start_cmd
+        cmd = self.start_cmd(node)
         self.logger.debug("VerifiableKafkaLog4jAppender %d command: %s" % (idx, cmd))
         node.account.ssh(cmd)
 
-    @property
-    def start_cmd(self):
-        cmd = "/opt/kafka/bin/kafka-run-class.sh org.apache.kafka.clients.tools.VerifiableLog4jAppender" \
-              " --topic %s --broker-list %s" % (self.topic, self.kafka.bootstrap_servers())
+    def start_cmd(self, node):
+        cmd = "/opt/%s/bin/" % kafka_dir(node)
```

Comment:
After seeing enough of these, I'm thinking in most cases you might actually want something more like `kafka_file(node, path)` which does something like `return os.path.join(kafka_dir(node), path)`. Just an idea since that'd also centralize a lot of the assumptions about `/opt`.